### PR TITLE
Dev1.1

### PR DESCRIPTION
**Pull request by ottojolanki**
Version 1.1

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,7 +53,8 @@ install_singularity: &install_singularity
     sudo make install
     singularity --version
 # Jobs
-version: 2
+orbs:
+  singularity: singularity/singularity@1.0.3
```
Comment (strattan) [inline .circleci/config.yml]: OK that this is different singularity version from what pipeline runs?

Comment (ottojolanki) [reply .circleci/config.yml]: 1.0.3 here refers to the version of the singularity orb. See more: https://circleci.com/orbs/registry/orb/singularity/singularity

File: workflow_opts/sherlock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-    "default_runtime_attributes" : {
-        "singularity_container" : "~/.singularity/rna-seq-pipeline-v1.0.simg",
-        "singularity_command_options" : "--bind /scratch,/lscratch,/oak/stanford"
-    }
-}
\ No newline at end of file
+  "default_runtime_attributes": {
+    "singularity_command_options": "--bind /scratch,/lscratch,/oak/stanford",
+    "singularity_container": "~/.singularity/rna-seq-pipeline-v1.0.simg"
```
Comment (strattan) [inline workflow_opts/sherlock.json]: Version bump needed here?

File: workflow_opts/singularity.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-    "default_runtime_attributes" : {
-        "singularity_container" : "~/.singularity/rna-seq-pipeline-v1.0.simg"
-    }
-}
\ No newline at end of file
+  "default_runtime_attributes": {
+    "singularity_container": "~/.singularity/rna-seq-pipeline-v1.0.simg"
```
Comment (strattan) [inline workflow_opts/singularity.json]: Version bump needed here?

File: workflow_opts/slurm.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
-      "slurm_partition": "YOUR_SLURM_PARTITON",
-      "slurm_account": "YOUR_SLURM_ACCOUNT",
-      "singularity_container" : "~/.singularity/rna-seq-pipeline-v1.0.simg",
-      "singularity_command_options" : "--bind /your/,[DATA_DIR1],[DATA_DIR2],..."
-    }
-}
\ No newline at end of file
+  "default_runtime_attributes": {
+    "singularity_command_options": "--bind /your/,[DATA_DIR1],[DATA_DIR2],...",
+    "singularity_container": "~/.singularity/rna-seq-pipeline-v1.0.simg",
```
Comment (strattan) [inline workflow_opts/slurm.json]: Version bump needed here?

File: workflow_opts/docker.json [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
-        "preemptible": "0",
-        "bootDiskSizeGb": "10",
-        "noAddress": "false"
-    }
-}
\ No newline at end of file
+  "default_runtime_attributes": {
+    "bootDiskSizeGb": "10",
+    "continueOnReturnCode": 0,
+    "docker": "quay.io/encode-dcc/rna-seq-pipeline:v1.0",
```
Comment (strattan) [inline workflow_opts/docker.json]: Version bump needed here?

File: docs/installation.md [hunk 6]
```diff
@@ -1,81 +1,53 @@
 # INSTALLATION
 
-To run the pipeline you need to install following software. Running the pipeline on Google Cloud or DNAnexus requires additional setup detailed below.
+To run the pipeline you need to do some setup. The exact steps you need to take depends on the platform you are running the pipeline on, and will be detailed below and in [HOWTO](howto.md). Independent of platform, running the pipeline is done using [caper](https://github.com/ENCODE-DCC/caper) and (optional but recommended) output organization is done using [croo](https://github.com/ENCODE-DCC/croo). Both `caper` and `croo` require `python` version 3.4.1 or newer.
```
Comment (strattan) [inline docs/installation.md]: Nit:  "steps ... depend on" to make number agree.

File: docs/reference.md [hunk 7]
```diff
@@ -45,19 +44,25 @@ Samtools flagstats are calculated using [Samtools 1.9](https://github.com/samtoo
 
 [bedGraphToBigWig kent source version 371](http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/bedGraphToBigWig) and [bedSort](http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/bedSort) (no version information available) are used in signal track generation. Source code is downloadable [here](http://hgdownload.soe.ucsc.edu/admin/exe/userApps.v371.src.tgz).
 
-## Genome Files
+## Genome Reference Files
+
+Reference and index files can be downloaded from the [ENCODE Portal](https://www.encodeproject.org/search/?type=Reference&reference_type=index). These files are for human, and mouse will follow. Files that are needed are STAR Index, RSEM Index, Kallisto Index, Chromosome sizes file. Links to required files are summarized in the table below.
```
Comment (strattan) [inline docs/reference.md]: Nit:  Chromosome->chromosome

File: .circleci/config.yml (hunk 1 above)
Comment (strattan) [reply .circleci/config.yml]: Ah OK

**Pull request merged**